docs/conf.py

```diff
@@ -0,0 +1,352 @@
+import os
+import re
+import json
+import subprocess
+import urllib.request
+from docutils import nodes
+
+from sphinx.util import console
+from sphinx.errors import SphinxError
+from recommonmark.parser import CommonMarkParser
+from recommonmark.transform import AutoStructify
+
+
+###########################################################################
+#                                                                         #
+#    Dredd documentation build configuration file                         #
+#                                                                         #
+###########################################################################
+
+
+# -- Environment ----------------------------------------------------------
+
+docs_dir = os.path.dirname(__file__)
+project_dir = os.path.join(docs_dir, '..')
+node_modules_bin_dir = os.path.join(project_dir, 'node_modules', '.bin')
+
+if os.environ.get('READTHEDOCS') == 'True':
+    installation_output = subprocess.getoutput('bash ' + os.path.join(docs_dir, 'install-node.sh'))
+    node_bin = installation_output.splitlines()[-1].strip()
+else:
+    node_bin = 'node'
+
+with open(os.path.join(project_dir, 'package.json')) as f:
+    package_json = json.load(f)
+
+
+# -- General configuration ------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'pygments_markdown_lexer',
+]
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+source_suffix = '.md'
+source_parsers = {'.md': CommonMarkParser}
+
+# The master document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Dredd'
+copyright = 'Apiary Czech Republic, s.r.o.'
+author = 'Apiary'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+def get_release():
+    try:
+        # Is internet available? this is to be able to generate docs
+        # e.g. in train without internet connection
+        urllib.request.urlopen('https://www.npmjs.com/package/dredd', timeout=3)
+    except urllib.request.URLError:
+        # Offline, use dummy release number
```

Comment:
Maybe for the case on ReadTheDocs (when `READTHEDOCS` is set) this should error to prevent the documentation from being stored and served with the wrong version.